<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ocelot 2.0</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>SZ3</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Dimension:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Error Bound:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Error Bound Mode</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ABS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>REL</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ABS_AND_REL</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Dataset Machine</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>98</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Machine A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string>Machine B</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dataset Directory:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>List Dataset</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Compress Selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Decompress Selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Transfer Selected</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Executable:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_21">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Executable</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>SZX</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>273</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>The Functionality is yet to be developed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ZFP</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>273</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>The Functionality is yet to be developed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Machine A Config</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>393</width>
      <height>322</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>FuncX ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="funcx_id_lineedit_a"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Globus ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="globus_lineedit_a"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Work Dir:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="workdir_lineedit_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="authenticate_button_a">
         <property name="text">
          <string>Authenticate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="list_workdir_button_a">
         <property name="text">
          <string>List Work Dir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_config_button_a">
         <property name="text">
          <string>Save Config</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_config_button_a">
         <property name="text">
          <string>Load Config</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="compress_button_a">
         <property name="text">
          <string>Compress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decompress_button_a">
         <property name="text">
          <string>Decompress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transfer_button_a">
         <property name="text">
          <string>Transfer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="auto_transfer_button_a">
         <property name="text">
          <string>Auto Transfer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="workdir_listwidget_a"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>411</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Machine B Config</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>393</width>
      <height>322</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>FuncX ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="funcx_id_lineedit_b"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Globus ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="globus_lineedit_b"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Work Dir:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="workdir_lineedit_b"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="authenticate_button_b">
         <property name="text">
          <string>Authenticate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="list_workdir_button_b">
         <property name="text">
          <string>List Work Dir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_config_button_b">
         <property name="text">
          <string>Save Config</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_config_button_b">
         <property name="text">
          <string>Load Config</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="compress_button_b">
         <property name="text">
          <string>Compress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decompress_button_b">
         <property name="text">
          <string>Decompress</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="transfer_button_b">
         <property name="text">
          <string>Transfer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="auto_transfer_button_b">
         <property name="text">
          <string>Auto Transfer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="workdir_listwidget_b"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Transfer Performance</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Current Status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
